src/index.css

```diff
@@ -38,6 +38,7 @@ limitations under the License.
   --quinary-content: #394049;
   --system: #21262c;
   --background: #15191e;
+  --backgrouund-85: rgba(23, 25, 28, 0.85);
```

Comment:
This should be accompanied by an environment variable in .env.example and src/main.tsx